test/fw/ptl/utils/pbs_testsuite.py

```diff
@@ -828,6 +829,132 @@ def init_proc_mon(self):
                                         frequency=freq)
                 self._process_monitoring = True
 
+    def revert_pbsconf(self):
+        """
+        Revert the following about the pbs.conf file:
+        - start bits
+        - PBS_SERVER
+        - PBS_SCP
+        - PBS_CORE_LIMIT
+        """
+        primary_server = self.servers.values()[0]
+
+        # Revert server hosts' pbs.conf
+        for server in self.servers.values():
+            cmds_to_exec = []
+            dmns_to_start = 0
+            pbs_conf_filepath = self.du.get_pbs_conf_file(server.hostname)
+            pbs_conf_val = self.du.parse_pbs_config(server.hostname,
+                                                    pbs_conf_filepath)
+            if pbs_conf_val is None or len(pbs_conf_val) < 1:
+                raise ValueError("Could not parse pbs.conf on host %s" %
+                                 (server.hostname))
+
+            # Set all start bits
+            if (pbs_conf_val["PBS_START_SERVER"] != "1"):
+                pbs_conf_val["PBS_START_SERVER"] = "1"
+                dmns_to_start += 1
+                cmds_to_exec.append(["server", "start"])
+            if (pbs_conf_val["PBS_START_SCHED"] != "1"):
+                pbs_conf_val["PBS_START_SCHED"] = "1"
+                cmds_to_exec.append(["sched", "start"])
+                dmns_to_start += 1
+            if self.moms is None or len(self.moms) == 0:
+                if (pbs_conf_val["PBS_START_MOM"] != "1"):
+                    pbs_conf_val["PBS_START_MOM"] = "1"
+                    cmds_to_exec.append(["mom", "start"])
+                    dmns_to_start += 1
+            else:
+                pbs_conf_val["PBS_START_MOM"] = "0"
```

Comment:
I am sure we have discussed it, but can you help me jog my memory about why are we disabling mom when self.moms is set. Also, don't you want to append mom stop to cmds_to_exec?

Reply:
If self.moms is set, we are disabling the mom on the server host as the tests will pick up moms from the self.moms hosts. Does that make sense, or do you think that's a bad idea?

Reply:
But doesn't PTL guarantee a mom on the local host to all the tests? If we do this then many tests will start failing

Reply:
actually, earlier my code was first turning the local mom off, and then bringing it back up when it parsed the self.moms list. I'm going to just check for it first now and not bring it down unnecessarily